conduct.md

```diff
@@ -52,35 +52,35 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is key to both successful and sustainable code. However it's important that all communication between students and team members follows the three pillars of Respect, Trust, and Humility. In general, students will *respect* their peers, *trust* their contributions and thoughts, and practice *humility* in all aspects of communication, in person, on GitHub, in the Discord, and otherwise.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Team members communitcating in a professional and respectful manner. 
+- Team members should use words like "Please" and "Thank you".
+- Team members will allow other members and teams speak freely. 
```

Comment:
I think that rephrasing acceptable conduct to have the verb at the beginning of each bullet point sounds better and can be standardized. However, I'll leave that up to your group and the class in general to decide.

Currently, the structure of the statements is not consistent in verb tenses.

For example here is how one of these bullet points can be restructured (From the section below):
Instead of:
```
Team members communicate in a way which offends or disrespects others. 
```
could be rephrased to:
```
Communicating in a way which offends or disrespects others. 
```
In most of these cases it's assumed that the conduct applies to everyone in the class.